packages/blade/src/utils/dataAttribute/dataAttribute.web.ts

```diff
@@ -0,0 +1,6 @@
+const dataAttribute = (name: string, value: string | undefined): Record<string, string> => {
```

Comment:
maybe let's rename this file to `automationAttributes` or `metaAttributes` or something generic since there's no such thing as `data` attributes on RN

Reply:
`metaAttribute` sounds good.

Reply:
Renamed 👍